ios/LayoutReanimation/REASwizzledUIManager.mm

```diff
@@ -1,91 +1,91 @@
+            fromClass:(Class)newClass
 {
-  if (!_blockSetter) {
-    _blockSetter = true;
-
-    self.bridge = bridge;
-    [super setValue:bridge forKey:@"_bridge"];
-    [self setValue:[bridge.uiManager valueForKey:@"_shadowViewRegistry"] forKey:@"_shadowViewRegistry"];
-    [self setValue:[bridge.uiManager valueForKey:@"_viewRegistry"] forKey:@"_viewRegistry"];
-    [self setValue:[bridge.uiManager valueForKey:@"_nativeIDRegistry"] forKey:@"_nativeIDRegistry"];
-    [self setValue:[bridge.uiManager valueForKey:@"_shadowViewsWithUpdatedProps"]
-            forKey:@"_shadowViewsWithUpdatedProps"];
-    [self setValue:[bridge.uiManager valueForKey:@"_shadowViewsWithUpdatedChildren"]
-            forKey:@"_shadowViewsWithUpdatedChildren"];
-    [self setValue:[bridge.uiManager valueForKey:@"_pendingUIBlocks"] forKey:@"_pendingUIBlocks"];
-    [self setValue:[bridge.uiManager valueForKey:@"_rootViewTags"] forKey:@"_rootViewTags"];
-    [self setValue:[bridge.uiManager valueForKey:@"_observerCoordinator"] forKey:@"_observerCoordinator"];
-    [self setValue:[bridge.uiManager valueForKey:@"_componentDataByName"] forKey:@"_componentDataByName"];
-
-    _blockSetter = false;
-  }
+  Method originalMethod = class_getInstanceMethod(originalClass, originalSelector);
+  Method newMethod = class_getInstanceMethod(newClass, newSelector);
+  IMP originalImplementation = method_getImplementation(originalMethod);
+  IMP newImplementation = method_getImplementation(newMethod);
+  class_replaceMethod(originalClass, newSelector, originalImplementation, method_getTypeEncoding(originalMethod));
+  class_replaceMethod(originalClass, originalSelector, newImplementation, method_getTypeEncoding(newMethod));
 }
 
-- (void)_manageChildren:(NSNumber *)containerTag
-        moveFromIndices:(NSArray<NSNumber *> *)moveFromIndices
-          moveToIndices:(NSArray<NSNumber *> *)moveToIndices
-      addChildReactTags:(NSArray<NSNumber *> *)addChildReactTags
-           addAtIndices:(NSArray<NSNumber *> *)addAtIndices
-        removeAtIndices:(NSArray<NSNumber *> *)removeAtIndices
-               registry:(NSMutableDictionary<NSNumber *, id<RCTComponent>> *)registry
+- (void)reanimated_manageChildren:(NSNumber *)containerTag
+                  moveFromIndices:(NSArray<NSNumber *> *)moveFromIndices
+                    moveToIndices:(NSArray<NSNumber *> *)moveToIndices
+                addChildReactTags:(NSArray<NSNumber *> *)addChildReactTags
+                     addAtIndices:(NSArray<NSNumber *> *)addAtIndices
+                  removeAtIndices:(NSArray<NSNumber *> *)removeAtIndices
+                         registry:(NSMutableDictionary<NSNumber *, id<RCTComponent>> *)registry
 {
   bool isLayoutAnimationEnabled = reanimated::FeaturesConfig::isLayoutAnimationEnabled();
   id<RCTComponent> container;
   NSArray<id<RCTComponent>> *permanentlyRemovedChildren;
   BOOL containerIsRootOfViewController = NO;
+  RCTUIManager *_self = (RCTUIManager *)self;
```

Comment:
Is it customary to use `_self` in this case? Maybe let's rename this field to something like `originalSelf` so that it's easier to see the difference?